specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/dscNodeConfiguration.json

```diff
@@ -189,12 +194,6 @@
               "$ref": "./definitions.json#/definitions/DscNodeConfiguration"
             }
           },
-          "201": {
```

Comment:
Why was this deleted? This may result in a breaking change in the generated clients.

Reply:
It returns only 200 and not 201. As mentioned earlier, we don't have any swagger-based stable clients and are in the process of releasing them.

I'll confirm the response code with Fiddler.

Reply:
It returns only 200 and not 201. As mentioned earlier, we don't have any swagger-based stable clients and are in the process of releasing them.

I'll confirm the response code with Fiddler.